react-app/package.json

```diff
@@ -23,7 +23,7 @@
     "lottie-react-native": "^5.1.4",
     "react": "18.2.0",
     "react-dom": "18.2.0",
-    "react-native": "0.71.4",
+    "react-native": "0.71.6",
```

Comment:
May I ask why did you change the version of react-native?

Reply:
It's recommended by expo. When using react-native 0.71.4, running `npx expo start` to start the preview locally, you will see the prompt.